tests/playwright/specs/a11y/includes/urls.js

```diff
@@ -8,7 +8,7 @@
 
 /**
  * URL paths for inclusion in page-level a11y scans.
- * Pages will be scanned at both desktop and mobile resolutions.
+ * Different set of pages will be scanned at desktop and mobile resolutions.
  */
 const desktopTestURLs = [
     '/en-US/',
```

Comment:
BTW I've changed the wording because I wasn't entirely sure what list would be used for what, so I hopefully got it right this way. Feel free to suggest any improvements if I'm not making it clear, or getting it completely wrong.